lib/seek/bio_schema/resource_decorators/creative_work.rb

```diff
@@ -25,7 +25,7 @@ def license
 
         def all_creators
           others = other_creators&.split(',')&.collect(&:strip)&.compact || []
-          others = others.collect { |name| { "@type": 'Person', "name": name } }
+          others = others.collect { |name| { "@type": 'Person', "@id": name.downcase.gsub(/[^0-9a-z]/i, '_'), "name": name } }
```

Comment:
I think you should use proper URI-encoding here, since its fairly common to have names with non-ASCII characters. Also the ID should be prefixed with `#`

Reply:
OK.

Reply:
You can do this using the ROCrate library: `"##{ROCrate::Entity.format_id(name)}"`

Reply:
Done